tox.ini

```diff
@@ -1,7 +1,21 @@
 [tox]
 envlist = py27-django{18,19,110,111}
 
+# This is needed to prevent the lms, cms, and openedx packages from getting
+# installed into site-packages where they can get imported, which is bad
+# because those won't even contain most of the source code.
```

Comment:
Mentioning the Open_edX package explicitly and maybe the top-level setup.py would be nice here.